src/store/reducers/food.js

```diff
@@ -0,0 +1,19 @@
+import { ADD_FOOD, DELETE_FOOD } from '../actions/food';
+
+let initState = {
+  foods: [],
+};
+
+const foodReducer = (state = initState, action) => {
+  switch (action.type) {
+    case ADD_FOOD:
+      return { ...state, foods: [...state.foods, action.payload] };
+    default:
```

Comment:
In reducer default option should always be the last one, because when you have multiple reducers like in this case that can be a problem. When dispatching an acton, redux doesn't now where to send action, so action goes throughout all reducers until he finds the right one.  So if reducer he is going through is not the right one state should be unchanged. That is the main usage of default option